src/components/DetailsMetrics/detailsMetrics.util.js

```diff
@@ -132,27 +132,46 @@ export const getDateRangeBefore = range => {
 const getMetricTitle = fullName =>
   fullName.substring(fullName.lastIndexOf('.') + 1).replace(/-/g, ' ')
 
-const timeFormatters = {
-  hours: {
-    formatMetricsTime: dates => {
-      const options = {
-        hour: 'numeric',
-        minute: TWO_DIGIT,
-        hour12: true
-      }
-      return dates.map(([date]) => new Date(date).toLocaleTimeString('en-US', options))
-    }
-  },
-  days: {
-    formatMetricsTime: dates => {
-      const options = {
-        year: TWO_DIGIT,
-        month: TWO_DIGIT,
-        day: TWO_DIGIT
-      }
-      return dates.map(([date]) => new Date(date).toLocaleDateString('en-US', options))
+const formatMetricsTime = (timeUnit, dates) => {
+  const timeFormatters = {
```

Comment:
no need to have `timeFormatters` inside the function

Reply:
done